<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>基本状态</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>301</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>击发状态</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditBullet">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>101</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>高低转速</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>261</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>枪塔归零</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHSpeed">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>32768</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>21</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>方位转速</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVSpeed">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>32768</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>221</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>余弹数量</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>61</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>方位转角</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>181</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>开栓状态</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHAngle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>32768</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>141</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>高低转角</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVAngle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>32768</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxGun">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0-无归零命令</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1-正在归零</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2-已归零</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxTrigger">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0-点射</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1-连射</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>231</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>按键码</string>
    </property>
    <widget class="QPushButton" name="pushButton_01">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>校枪使能（01）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_02">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>校枪失能（02）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_04">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>手柄失能（04）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_03">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>手柄使能（03）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_06">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>弹药重装（06）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_05">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开栓（05）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_08">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关机（08）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_07">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>枪塔归零（07）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0a">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>下移（0a）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_09">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>上移（09）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0c">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>右/取消（0c）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0b">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>左/确认（0b）</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>反馈</string>
    </property>
    <widget class="QLabel" name="labelState">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
